_community_members/jiapingzeng.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Jiaping Zeng
+short_name: Jiaping
+photo: "/assets/media/community/members/jiapingzeng.jpg"
+title: 'OpenSearch Community Member: Jiaping Zeng'
+primary_title: Jiaping Zeng
+breadcrumbs:
+  icon: community
+  items:
+    - title: Community
+      url: /community/index.html
+    - title: Members
+      url: /community/members/index.html
+    - title: 'Jiaping Zeng&apos;s Profile'
+      url: '/community/members/jiapingzeng.html'
+github: arjunkumargiri
+job_title_and_company: 'Software engineer at Amazon Web Services'
+personas:
+  - author
+permalink: '/community/members/jiapingzeng.html'
+redirect_from: '/authors/jiapingzeng/'
+---
+
+**Jiaping Zeng** is a Software Engineer at AWS working on the OpenSearch Project. He primarily works on OpenSearch's AI/ML and semantic search feature. He is passionate about AI, ML and building scalable systems.
```

Comment:
@arjunkumargiri I'm assuming this is a placeholder bio because it's the same as yours. Please update this bio before we publish the blog.